packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+// @ts-ignore
+import RCCascader from 'rc-cascader';
+
+export interface ButtonCascaderOption {
+  label: string;
+  value: string;
+
+  children?: ButtonCascaderOption[];
```

Comment:
Not sure about `children` name, as it automatically resembles React's children property. What about `items`?